plugins/prune-unused.yaml

```diff
@@ -26,7 +26,8 @@ spec:
         kubectl prune-unused <configmaps|secrets> [options]
 
     Options:
```

Comment:
Please remove usage from here, as it will always go out of date. It's not necessary of you support -h/--help.